blurring_as_a_service/inference_pipeline/components/detect_and_blur_sensitive_data.py

```diff
@@ -29,9 +30,10 @@
 def detect_and_blur_sensitive_data(
     mounted_root_folder: Input(type=AssetTypes.URI_FOLDER),  # type: ignore # noqa: F821
     relative_paths_files_to_blur: Input(type=AssetTypes.URI_FILE),  # type: ignore # noqa: F821
-    model: Input(type=AssetTypes.URI_FOLDER),  # type: ignore # noqa: F821
     yolo_yaml_path: Output(type=AssetTypes.URI_FOLDER),  # type: ignore # noqa: F821
     results_path: Output(type=AssetTypes.URI_FOLDER),  # type: ignore # noqa: F821
+    customer_name: str,
```

Comment:
shouldn't this be an Input/Output type

Reply:
I don't know what type=AssetTypes I need to define then. There is no option to use a string type, or I need to write the string to a file.

Reply:
If it's just a string (and no Input or Output folder) it's possible to use the type str